bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Zh_M125_ToAA_Tobbgg_LO_MLM/Zh_M125_ToAA__AMASS__Tobbgg_customizecards.dat

```diff
@@ -0,0 +1,7 @@
+set param_card mzdinput _ZDMASS_
+set param_card mhsinput _AMASS_
+set param_card epsilon 1.000000e-10
+set param_card kap 5.000000e-01
+set param_card whs 5.237950e-03
+set param_card mass 35 _AMASS_
+set param_card mass 1023 _ZDMASS_
```

Comment:
Sorry, forgot to comment. Do you not change the widths of A and ZD?

Reply:
We are changing the A mass from 20 to 60 GeV, that is already implemented in a python code.. But ZD mass is not necessary to us..

Reply:
What about the widths of the particles, not the mass.

Reply:
Hi @sihyunjeon , we are not changing the width of A, and following the H-->AA--> 4gamma analysis startegy. Reference is here : https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/hToaaTo4gamma_ma_AMASS_GeV_MLM_4f_max1j/hToaaTo4gamma_ma_AMASS_GeV_MLM_4f_max1j_customizecards.dat#L5

Reply:
Then do you get your cross sections from the theorists or any theory paper? If you set the mass in such manner, the cross sections you get from madgraph will be totally wrong.

Reply:
Hi, @sihyunjeon, we have already checked that, the MadGraph cross-section is not reliable. So, we already have a theoretical motivation and we can get the actual cross-section from a theory paper.

Reply:
@Saptaparna @GurpreetSinghChahal Please check and merge